contrib/systemd/system/zfs-import-cache.service.in

```diff
@@ -4,10 +4,11 @@ DefaultDependencies=no
 Requires=systemd-udev-settle.service
 After=systemd-udev-settle.service
 After=cryptsetup.target
-ConditionPathExists=@sysconfdir@/zfs/zpool.cache
+ConditionPathExists=${ZFS_CACHE}
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 ExecStartPre=/sbin/modprobe zfs
-ExecStart=@sbindir@/zpool import -c @sysconfdir@/zfs/zpool.cache -aN
+ExecStart=@sbindir@/zpool import ${ZFS_CACHE:+-c "${ZFS_CACHE}"} -aN ${ZPOOL_IMPORT_OPTS}
+EnvironmentFile=-@initconfdir@/zfs
```

Comment:
Neat, I didn't know this existed but it's a nice approach.  Just one question, why `=-` instead of `=`?